packages/replay/test/unit/coreHandlers/handleFetch.test.ts

```diff
@@ -1,6 +1,5 @@
-import { mockSdk } from '@test';
-
 import { handleFetch } from '../../../src/coreHandlers/handleFetch';
+import { mockSdk } from './../..';
```

Comment:
I'm just wondering about this path, wouldn't you normally have to specify the file?

Reply:
Hmm yeah, should we make it `./../../index`? WDYT? (Guess this applies at all places) - maybe a bit more explicit...!